pkg/admin/services/provider_service_impl_test.go

```diff
@@ -0,0 +1,271 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package services
+
+import (
+	"net/url"
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/apache/dubbo-admin/pkg/admin/util"
+
+	"github.com/apache/dubbo-admin/pkg/admin/constant"
+
+	"github.com/apache/dubbo-admin/pkg/admin/cache"
+
+	"dubbo.apache.org/dubbo-go/v3/common"
+
+	"github.com/apache/dubbo-admin/pkg/admin/model"
+)
+
+var testProvider *model.Provider
+
+func init() {
```

Comment:
Is there a way to clear the cache? Need to make sure it will not affect tests in other files.